Library/Formula/fig.rb

```diff
@@ -83,6 +83,6 @@ def install
 
   test do
     output = shell_output(bin/"docker-compose --version")
-    assert output.include? "compose 1.2.0"
+    assert output.include? "compose 1.3.0"
```

Comment:
I think these two lines can be changed to just `assert_match version.to_s shell_output(bin/"docker-compose --version")`